### DISCUSSION TITLE
database里的数据复制到另一个database

### DISCUSSION THREAD

**Discussion by chaixingyao-code** [taosdata/TDengine#4710] [General]
我有一个test1数据库，里面有几张表，表里有数据。
我想把里面的数据，全部复制到test2库里，请问怎么能做到呢？？
服务器版本:TDengine-server-2.0.10.0-Linux-x64.rpm

**Comment by chaixingyao-code** [comment]
把test1里面的数据，复制一份到test2里面，这个能实现么

**Comment by sangshuduo** [comment]
taosdump dbname -o <dir>

bash:
for file in `ls <dir>`; do sed -i 's/test/test2/g' $file; done

taosdump -i <dir>
